setup.py

```diff
@@ -61,6 +61,7 @@
           ],
           extras_require={
               'test': [
+                  'mne_qt_browser',
```

Comment:
looks like we need to add `PyQt5` as a dependency too since the mne_qt_browser could not find Qt bindings

Reply:
true, thanks for reminding me :-) pushed a commit!